### ISSUE TITLE
Places where French is missing

### ISSUE THREAD

**Issue by RachelMuston** [canada-ca/devex#8]
Message when users sign up for gov access.
![govuservalidation_devex](https://user-images.githubusercontent.com/29108013/33246549-b974328c-d2e3-11e7-890c-2391997d9681.png)

**Comment by markwooff** [comment]
Do we have translated text for this? I can place it in the site if we do.

**Comment by markwooff** [comment]
Actually if we could get translations for the following, it will cover all outcomes from the Edit Profile screen:

- 'Edit profile successful',
- 'You have requested government user access, the request is now posted for review. You will receive the government access and be able to access gov user functionality as soon as the admin verifies you as government user.',
- 'We will send you notifications of new Code With Us Opportunities.',
- 'We will notify you of upcoming events.',
- 'We will notify you of new blog posts.',
- 'Edit profile failed!',
